[AUTOCUT] OS Distribution Build Failed for security-2.8.0

***Received Error***: **Error building security, retry with: ./build.sh manifests/2.8.0/opensearch-2.8.0.yml --component security**.
                      The distribution build for security has failed.
                      Please see build log at https://build.ci.opensearch.org/job/distribution-build-opensearch/7813/consoleFull

```
> Task :compileJava
/tmp/tmpflq2uyli/security/src/main/java/org/opensearch/security/transport/SecurityRequestHandler.java:334: error: getExtensionIdMap() is not public in ExtensionsManager; cannot be accessed from outside package
            if (extManager.getExtensionIdMap().containsKey(extensionUniqueId)) {
                          ^
[  2%] Building CXX object external/faiss/faiss/CMakeFiles/faiss.dir/Clustering.cpp.o
[ 55%] Building CXX object external/faiss/faiss/CMakeFiles/faiss.dir/index_factory.cpp.o
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
```

--- a/src/main/java/org/opensearch/security/transport/SecurityRequestHandler.java
+++ b/src/main/java/org/opensearch/security/transport/SecurityRequestHandler.java
@@ -331,7 +331,7 @@ public class SecurityRequestHandler<T extends TransportRequest> extends Security
         String extensionUniqueId = getThreadContext().getHeader("extension_unique_id");
         if (extensionUniqueId != null) {
             ExtensionsManager extManager = OpenSearchSecurityPlugin.GuiceHolder.getExtensionsManager();
-            if (extManager.getExtensionIdMap().containsKey(extensionUniqueId)) {
+            if (extManager.lookupExtensionSettingsById(extensionUniqueId).isPresent()) {
                 getThreadContext().putTransient(ConfigConstants.OPENDISTRO_SECURITY_SSL_TRANSPORT_EXTENSION_REQUEST, Boolean.TRUE);
             }
         }

